community/btrbk/APKBUILD

```diff
@@ -1,27 +1,25 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=btrbk
-pkgver=0.28.0
+pkgver=0.28.2
 pkgrel=0
 pkgdesc="Backup tool for btrfs subvolumes with atomic snapshots & incremental transfers"
 url="https://digint.ch/btrbk"
 arch="noarch"
 license="GPL-3.0-or-later"
 depends="perl perl-date-calc btrfs-progs"
-makedepends="asciidoc"
+makedepends="asciidoctor"
 subpackages="$pkgname-doc"
 source="https://digint.ch/download/btrbk/releases/$pkgname-$pkgver.tar.xz"
 builddir="$srcdir"/$pkgname-$pkgver
```

Comment:
```suggestion

```

Reply:
The `APKBUILD` already has:

`builddir="$srcdir"/$pkgname-$pkgver`